exonum/src/storage/db.rs

```diff
@@ -13,7 +13,8 @@
 // limitations under the License.
 
 use std::collections::HashMap;
-use std::collections::btree_map::{BTreeMap, Range};
+use std::collections::btree_map::{BTreeMap, Range, Iter as BTMIter, IntoIter as BTMIntoIter};
+use std::collections::hash_map::{Iter as HMIter, IntoIter as HMIntoIter, Entry as HMEntry};
```

Comment:
I feel like these names are more close to the Rust naming conventions (upper camelcase for types, https://doc.rust-lang.org/1.0.0/style/style/naming/README.html): BtmIter, BtmIntoIter, HmIter, HmIntoIter, HmEntry. What do you think?